test/F1Register_test.js

```diff
@@ -166,9 +164,10 @@ describe('F1Register', function() {
     }
   })
 
-  describe('searchQuery', function() {
-    it('yields query based on valid registration object', function(done) {
-      f1reg.searchQuery(sub, function(err, query) {
+  describe('searchQuery', function () {
+    it('yields query based on valid registration object', function (done) {
+      f1reg.searchQuery(sub, function (err, query) {
+        should(err).not.exist
```

Comment:
Expected an assignment or function call and instead saw an expression.<br>Missing semicolon.